src/index.html

```diff
@@ -7,6 +7,50 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="sparkline">
+      <div class="sparkline__day">mon</div>
+      <div class="sparkline__day">tue</div>
+      <div class="sparkline__day">wed</div>
+      <div class="sparkline__day">thu</div>
+      <div class="sparkline__day">fri</div>
+      <div class="sparkline__day">sat</div>
+      <div class="sparkline__day">sun</div>
+    </div>
+
+    <div class="calendar calendar--month-length_31">
```

Comment:
Rename modifiers following BEM naming convention for complex modifiers: `block-name--modifier-name--modifier-value` (checklist)